### ISSUE TITLE
Connected (Glass) Textures in 1.13.1 & Pre-7 (Solved)

### ISSUE THREAD

**Issue by chland** [sp614x/optifine#1913]
I don't know if this is a bug or a problem with the resource packs i tested. I used two different "Clear Glass" Packs. I changed the "mcpatcher"-folder to "optifine" and it seems like the Connected Textures are "working". But I'm experiencing a strange problem. If i look through glass towards the west or north, the glass looks as expected but if I look through the same glass-panes towards the south or east, CTM doesn't seem to work properly. Here is a little screenshot:

![connected](https://user-images.githubusercontent.com/472894/47026426-b0a05600-d165-11e8-8005-81ce6fdd3aab.jpg)

The "glass_pane.properties" looks like this:

```
matchBlocks=glass_pane
connect=block
method=ctm
faces=sides
tiles=0-47
```

I also tried `faces=north east south west` with the same result.

**Comment by D3monPixel** [comment]
That is a problem with the glass pane model. Vanilla just has it modeled in some fast way without mirroring the texture to the other side. That mirroring is required though for the ctm to work correctly on them. 
So you'll have to fix up the block model templates for glass panes in your pack.

**Comment by chland** [comment]
Ah, thanks for the feedback, guys! That actually helped a lot.

I did some tests and it turned out that it was caused by another pack. I'm using the Texture-pack that the Hermitcraft-peeps released (basically the new Beta-textures ported to 1.13 -> https://www.reddit.com/r/HermitCraft/comments/90e4du/for_those_of_you_asking_about_the_texture_pack/) and it looks like the combination of a clear-glass pack with the "beta-pack" messes up the connected textures as the "beta-pack" contains a ton of model-files for all kinds of glass. I removed those models from the beta-pack and now everything works fine.

Only strange thing is that I'm at least 75% certain that the same combination worked in Pre1-Pre4 (?).

Anyways - sorry for wasting your time :)
